<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HNS_Home_Screen</class>
 <widget class="QWidget" name="HNS_Home_Screen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1140</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="PB_Home_NewSchedule">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>620</y>
     <width>250</width>
     <height>83</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#1A3760;
    color:#FFFFFF;
	border-radius:10px</string>
   </property>
   <property name="text">
    <string>CREATE NEW
SCHEDULE</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="GV_Home_Logo">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>10</y>
     <width>52</width>
     <height>52</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Home_EditMessage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>60</y>
     <width>250</width>
     <height>83</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#1A3760;
    color:#FFFFFF;
	border-radius:10px</string>
   </property>
   <property name="text">
    <string>EDIT CURRENT
MESSAGE</string>
   </property>
  </widget>
  <widget class="QLabel" name="LBL_Home_Page">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>500</y>
     <width>250</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Pages 1</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>15</y>
     <width>721</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>33</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#1A3760</string>
   </property>
   <property name="text">
    <string>CURRENT MESSAGE BOARD</string>
   </property>
  </widget>
  <widget class="HNS_Signboard_Preview" name="GV_HomePreview" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>695</width>
     <height>443</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Home_StopMessage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>420</y>
     <width>250</width>
     <height>83</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#1A3760;
    color:#FFFFFF;
	border-radius:10px</string>
   </property>
   <property name="text">
    <string>STOP
MESSAGE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="PB_Home_CreateMessage">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>150</y>
     <width>250</width>
     <height>83</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lato</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">    background-color:#1A3760;
    color:#FFFFFF;
	border-radius:10px</string>
   </property>
   <property name="text">
    <string>CREATE NEW
MESSAGE</string>
   </property>
  </widget>
  <widget class="HNS_Radar_Info_Widget" name="PGE_Radar_Info" native="true">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>240</y>
     <width>280</width>
     <height>157</height>
    </rect>
   </property>
  </widget>
  <widget class="HNS_Schedule_Info_Widget" name="PGE_Current_Schedule" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>700</width>
     <height>155</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HNS_Signboard_Preview</class>
   <extends>QWidget</extends>
   <header>hns_signboard_preview.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Radar_Info_Widget</class>
   <extends>QWidget</extends>
   <header>hns_radar_info_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HNS_Schedule_Info_Widget</class>
   <extends>QWidget</extends>
   <header>hns_schedule_info_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
